DEVELOP_V2.md

```diff
@@ -119,8 +119,8 @@ To change the `nrjmx` communication protocol, modify the `./commons/nrjmx.thrift
 make code-gen
 ```
 
-This command will generate the code for both: `gojmx` project and `nrjmx` project. Is running inside a docker container
-so no dependencies are required. Push the changes for the generated files on your branch to make sure `go get` command
+This command will generate the code for both: `gojmx` project and `nrjmx` project. Is running inside a 
+[docker container](commons/Dockerfile) so no dependencies are required. Push the changes for the generated files on your branch to make sure `go get` command
```

Comment:
while developing we should have access to the pushed thrift image, so no local build of the image should be required